# sequence ID: consecutively increasing ID
time stamp
float32[] position
float32[] velocity
float32[] effort
